fe/fe-core/src/main/java/com/starrocks/qe/StmtExecutor.java

```diff
@@ -991,7 +991,8 @@ private void handleUseDbStmt() throws AnalysisException {
     private void handleUseCatalogStmt() throws AnalysisException {
         UseCatalogStmt useCatalogStmt = (UseCatalogStmt) parsedStmt;
         try {
-            context.getGlobalStateMgr().changeCatalog(context, useCatalogStmt.getCatalogName());
+            String catalogName = useCatalogStmt.getCatalogAndItsName().split("\\s+")[1];
```

Comment:
You can put split logic into `UseCatalogStmt`.

Reply:
done